apps/login/ui/Info.tsx

```diff
@@ -0,0 +1,14 @@
+import { ExclamationTriangleIcon } from "@heroicons/react/24/outline";
```

Comment:
Is this not redundant with ui/Alert.tsx?

Reply:
yes. removed